src/PeriodEnd/Refunds/SFA.DAS.ProviderPayments.Calc.Refunds/Domain/RefundGroup.cs

```diff
@@ -13,7 +14,7 @@ public RefundGroup(RequiredPaymentEntity entity)
             ApprenticeshipContractType = entity.ApprenticeshipContractType;
         }
 
-        public RefundGroup(HistoricalPaymentEntity entity)
+        public RefundGroup(HistoricalPayment entity)
```

Comment:
check your renames, might be a few other var names that need updating like this one, from `entity` to something `historicalPayment` or similar.

Reply:
Good spot, missed that

Reply:
Actually, I just looked again. The parameter Type in the RefundGroup needs to use the domain object and not the entity from the refunds repo

Reply:
It was resharper which renamed these refs